lib/modules/dosomething/dosomething_kudos/dosomething_kudos.install

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @file
+ * Installation and schema hooks for dosomething_kudos.module
+ */
+
+/**
+ * Implements hook_schema().
+ */
+function dosomething_kudos_schema() {
+  $schema = array();
+  $schema['dosomething_kudos'] = array(
+    'description' => 'Table of kudos interactions on reportback items.',
+    'fields' => array(
+      'kid' => array(
+        'description' => 'The primary identifier for a kudos record.',
+        'type' => 'serial',
+        'unsigned' => TRUE,
```

Comment:
These should all be 
`unsigned => FALSE`
which is be the default already, so you can remove all the references to `unsigned`